### DISCUSSION TITLE
How to add images after project build?

### DISCUSSION THREAD

**Discussion by lfazliev** [nuxt/nuxt#21387] [Questions]
Can I add images to .output/public after building the project to get it by root url? If so, what settings do I need for my project? I am running it on a node.js server.

**Comment by fabianwohlfart** [comment]
Did you find an answer to that?

**Comment by fabianwohlfart** [comment]
I will put my use case to it:
We have a blog which is build and runs in a custom node env.
Now we add a new article to it with images. How do we display the new images?

**Comment by lfazliev** [comment]
At the moment, I found a solution in using nuxt image, it helps to pull images from public, after building the project.

**Reply by TLovers** [reply]
hi, I want to add some static resources to the output folder after output is generated. How can I listen to the build completion hook

**Comment by KeJunMao** [comment]
Do you mean [public](https://nuxt.com/docs/guide/directory-structure/public)?

Or you can use hooks: 
```ts
// nuxt.config.ts
import { copyFileSync } from 'node:fs'

export default defineNuxtConfig({
  hooks: {
    close() {
      copyFileSync('./package.json', './.output/public/package.json')
    },
  },
})

```

**Reply by TLovers** [reply]
thks

**Comment by TheAlexLichter** [comment]
Linking https://github.com/nuxt/nuxt/issues/19263 in favor.
